k9mail/src/main/java/com/fsck/k9/planck/ui/fragments/GoogleAuthGuideStep2Fragment.kt

```diff
@@ -30,6 +30,6 @@ class GoogleAuthGuideStep2Fragment : AccountSetupBasicsFragmentBase() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        navigator.setCurrentStep(AccountSetupNavigator.Step.GOOGLE_GUIDE_STEP_1, null)
+        navigator.setCurrentStep(AccountSetupNavigator.Step.GOOGLE_GUIDE_STEP_2, null)
```

Comment:
why do we start from step2?

Reply:
Just this is the step 2 fragment, so we are just setting the step we are in. I forgot to do it before, there was a copy-paste bug but that does not harm just because of the flow.
Just very old navigation still in place...